### ISSUE TITLE
No Checkbox Focus Indicator

### ISSUE THREAD

**Issue by hansl** [angular/material2#250]
There's one for radio button but not for checkbox.

**Comment by jelbourn** [comment]
cc @traviskaufman are you able to take a look at this? I believe the checkbox should have a focus indicator when the focus arrives via keyboard.
